plugins/generator-1.17.1/forge-1.17.1/templates/procedure.java.ftl

```diff
@@ -30,9 +30,11 @@
 <#-- @formatter:off -->
 package ${package}.procedures;
 
+<#if trigger_code?has_content>

```

Comment:
Keep this system as is and modify your templates instead. Check how I modified your first PR to work properly

Reply:
> 
> 
> Keep this system as is and modify your templates instead. Check how I modified your first PR to work properly

I think it's the rest of the bad merging I made. I thought it was removed.

Reply:
Can you try to refresh your page? I don't see something changed in the procedure template.

Reply:
Indeed my browser cached it :D